core/src/main/scala/org/apache/spark/internal/config/UI.scala

```diff
@@ -203,4 +203,15 @@ private[spark] object UI {
     .stringConf
     .transform(_.toUpperCase(Locale.ROOT))
     .createWithDefault("LOCAL")
+
+  val ENABLED_STREAMING_UI_CUSTOM_METRIC_LIST =
```

Comment:
I see the SS UI relevant configurations are in StaticSQLConf - we can move the configuration and rename to the similar.

Reply:
Please see my previous comment above. `SQLConf` work only of there is an active session, otherwise the config not propagated but silently gives back the default. Since in the webserver there is no active session (because it's common) I don't think we want to itroduce such. Is `StaticSQLConf` propagated w/o session?

Reply:
I've just had a deeper look at `StaticSQLConf` which works when it's read from `SparkConf`. We can do the exact same approach like `spark.sql.streaming.ui.retainedQueries` but I think it can't be really unit tested (at least not found a way to do).

Reply:
After several tests done it looks like `StaticSQLConf` is the right solution (though reaching the value not w/ `SQLConf.get` but w/ `SparkConf`) which fits into the existing config set. Thanks for notifying, this way much better and unit test works also.